### DISCUSSION TITLE
Have we discussed a Py5Image objects .copy() issue once?

### DISCUSSION THREAD

**Discussion by villares** [py5coding/py5generator#442] [Q&A] [answered]
I remember that Py5Graphics objects are copied as Py5Image objects, and that's it.
Then, I kind of remember talking about this... But I couldn't find an open issue.

```python
>>> img
Py5Image(width=640, height=384)
>>> img.copy()
<java object 'processing.core.PImage'>
```
I think you told me it was an easy fix, maybe?
I'm not sure anymore. The image came from a `load_image()` call.

**Comment by hx2A** [comment]
> I think you told me it was an easy fix, maybe?

Silly me! I made this fix for Py5Graphics but didn't think to check if Py5Image also had a `copy()` method. Apparently it does.

This is a bug, and is an easy one to fix. I'll open an issue for it.

**Reply by hx2A** [reply, marked as answer]
Fixed! Thanks for spotting this, @villares !
